qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -98,7 +85,7 @@ def level_0_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassMa
         "scheduling", scheduling_method, pass_manager_config, optimization_level=0
     )
 
-    init = common.generate_control_flow_options_check(
+    pre_init = common.generate_control_flow_options_check(
```

Comment:
The whole concept of a pre-initialisation stage is funny to me, but this does seem like an ok compromise.